R/unit.conversions.R

```diff
@@ -164,7 +168,7 @@ generateUnitsData <- function() {
 #' validMetadataUnits("g", unit.type="flow.units") # FALSE
 validMetadataUnits <- function(unitstr, unit.type=c("ANY","flow.units","conc.units","load.units","load.rate.units")) {
   unit.type <- match.arg(unit.type)
-  
+  unit=NULL
```

Comment:
this is fixed here https://github.com/miguelcleon/loadflex/blob/master/R/unit.conversions.R#L178